### DISCUSSION TITLE
Blog tanpa CMS?

### DISCUSSION THREAD

**Discussion by adhemukhlis** [adhemukhlis/me#21] [Blogs]
# TITLE 1
## TITLE 2

``` js
const any_variable = 'exist'
```

> hello!

**Bold**

*Italic*

~~Strikethrough~~

Text<sub>Subscript</sub>

Text<sup>Superscript</sup>

<ins>Underline</ins>

how about `react`?

visit to my personal website [adhemukhlis.vercel.app](https://adhemukhlis.vercel.com/)

![Screenshot of a comment on a GitHub issue showing an image, added in the Markdown, of an Octocat smiling and raising a tentacle.](https://myoctocat.com/assets/images/base-octocat.svg)

LIST:
- first
* second
+ third

numbered list
1. James Madison
2. James Monroe
3. John Quincy Adams

nested list
1. First list item
   - First nested list item
     - Second nested list item

TODO:
- [ ] (Optional) Open a followup issue

> [!NOTE]
> Useful information that users should know, even when skimming content.

> [!TIP]
> Helpful advice for doing things better or more easily.

> [!IMPORTANT]
> Key information users need to know to achieve their goal.

> [!WARNING]
> Urgent info that needs immediate user attention to avoid problems.

> [!CAUTION]
> Advises about risks or negative outcomes of certain actions.

| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |

| Left-aligned | Center-aligned | Right-aligned |
| :---         |     :---:      |          ---: |
| git status   | git status     | git status    |
| git diff     | git diff       | git diff      |

<details>

<summary>Tips for collapsed sections</summary>

### You can add a header

You can add text within a collapsed section. 

You can add an image or a code block, too.

```ruby
   puts "Hello World"
```

</details>

Here is a simple flow chart:

```mermaid
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```

**Comment by adhemukhlis** [comment]
comment test

**Reply by adhemukhlis** [reply]
reply test

**Reply by awastebank** [reply]
reply +1

**Comment by adhemukhlis** [comment]
comment test (2)

**Reply by awastebank** [reply]
first reply

**Comment by awastebank** [comment]
comment test (3)
